src/owlwatch/schema/model.py

```diff
@@ -264,10 +274,16 @@ def from_json(cls, index_name, mapping_json):
                         prop_name = prop + '.' + nested_prop
                         analyzed = None
                         agg = None
+
                         if 'fielddata' in nested_value:
                             agg = nested_value['fielddata']
+                        if 'type' in nested_value:
+                            ptype = nested_value['type']
+                        else:
+                            print('Not adding to es_mapping value: %s' % nested_value)
```

Comment:
As this is (intends to be) a library, we should use logging for writing messages. I'll also explain why we are not adding that property in the message.